### PR TITLE
Reference ET equation

### PR DESCRIPTION
**Pull request by mamatapandey**
The file consists of Penman Monteith equation for estimating Reference ET.

### CODE REVIEW

File: Reference_ET_Part_1.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+# Saturation  Vapour Pressure at Min temp
+e_tmin = 0.6108*math.exp(17.27*t_min/(t_min+237.3))
+
+# Taking average of sat vapour pressure at max and min temperature
+es = np.mean([e_tmax, e_tmin])
+
+# Calculating Actual Vapour Pressure
+
+regex = '^(0|[1-9]\d*)(\.\d+)?$'
```
Comment (jsadler2) [inline Reference_ET_Part_1.py:34]: Can you write a comment here explaining this regex? I don't know what that means.

File: Reference_ET_Part_1.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+# Determining vapour pressure deficit(es_ea)
+ 
+es_ea = round(es - ea, 3)
+
+## Extraterrestrial Radiation
+
+month  = 
+day = 
+year =
```
Comment (jsadler2) [inline Reference_ET_Part_1.py:56]: looks like you need to add something here?

**Review by jsadler2 — COMMENTED** [review]
I've left a couple of comments

File: Reference_ET_Part_1.py (hunk 1 above)
Comment (mamatapandey) [inline Reference_ET_Part_1.py:34]:
```suggestion
# The expression is used to check if there is some value in the humidity data. 

regex = '^(0|[1-9]\d*)(\.\d+)?$'
```

**Pull request merged**